src/ui/server/helpers/generate-change-link/index.ts

```diff
@@ -13,7 +13,18 @@ const { INSURANCE_ROOT } = ROUTES.INSURANCE;
  * @param {Boolean} checkAndChange true if coming from check your answers section in submit application section
  * @returns
  */
-const generateChangeLink = (route: string, routeCheckAndChange: string, anchorTag: string, referenceNumber: number, checkAndChange: boolean) => {
+const generateChangeLink = (
+  route: string,
+  routeCheckAndChange: string,
+  anchorTag: string,
+  referenceNumber?: number,
+  checkAndChange?: boolean,
+  isInsuranceEligibility = false,
+) => {
+  if (isInsuranceEligibility) {
+    return `${route}${anchorTag}`;
+  }
+
```

Comment:
Can we add some documentation about this please, I.e the insurance eligibility routes already contain `INSURANCE_ROOT`

Maybe we should change this so the routes don't have this 🤔 Not for this PR though